stories/tabs.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Tabs from '../src/web/components/Tabs';
+import Text from '../src/web/components/Text';
+
+const data = [
+  { key: 'Hello', component: () => <Text>Hello</Text> },
+  { key: 'Bonjour', component: () => <Text>Bonjour</Text> },
+  { key: 'Ciao', component: () => <Text>Ciao</Text> },
+];
+
+storiesOf('Tabs', module)
+  .add('Basic Tabs', () => <Tabs data={data} onChange={action('Tabs')} />)
+  .add('with size', () => [
+    <Tabs data={data} size="small" onChange={action('small')} />,
+    <Tabs data={data} onChange={action('default')} />,
```

Comment:
When we select a tab, a part of the screen should display the `component: ()`, it's not the case in the storybook

Reply:
My bad when you say :
```
const tabs = [{
 key: 'Hello',
 component: () => <SomeComponent />
}]
``` 
I thought the component field would be the content of the onglet.
I'm going to update it.